wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,78 @@
+# How to install drivers and fix them 
+Hello, in this post you will know how to download or update your drivers for your computer.
+It's highly recommended to not use third-party software to update drivers.
```

Comment:
I mean, i guess this is not really needed but it sounds a bit more fluent to me like this :

It's highly recommended to avoid using third-party software to update or install drivers.